api/src/main/java/net/kyori/adventure/text/format/Style.java

```diff
@@ -38,7 +38,18 @@
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
- * A style.
+ * Something that adds functionality to plain text,
+ * be it adding {@link TextColor}s, {@link TextDecoration}s, {@link ClickEvent}s etc.
```

Comment:
```suggestion
 * A style applies visual effects or extra functionality to {@link Component}s,
 * such as {@link TextColor}s, {@link TextDecoration}s, {@link ClickEvent}s etc.
```

This should probably also have a list of `@see`s?